app/src/main/java/app/revanced/manager/ui/component/bundle/BundlePatchesDialog.kt

```diff
@@ -62,44 +61,204 @@ fun BundlePatchesDialog(
+                            verticalArrangement = Arrangement.spacedBy(4.dp)
+                        ) {
+                            PatchInfoChip(
+                                modifier = Modifier.align(Alignment.CenterVertically),
+                                text = "\uD83D\uDCE6 $packageName"
+                            )
+
+                            if (versions.isNotEmpty()) {
+                                if (expandVersions) {
+                                    versions.forEach { version ->
+                                        PatchInfoChip(
+                                            modifier = Modifier.align(Alignment.CenterVertically),
+                                            text = "\uD83C\uDFAF $version"
+                                        )
+                                    }
+                                } else {
+                                    PatchInfoChip(
+                                        modifier = Modifier.align(Alignment.CenterVertically),
+                                        text = "\uD83C\uDFAF ${versions.first()}"
+                                    )
+                                }
+                                if (versions.size > 1) {
+                                    PatchInfoChip(
+                                        onClick = onExpandVersions,
+                                        text = if (expandVersions) "Less" else "+${itemCount - 2}"
                                     )
                                 }
                             }
+                        }
+                    }
+                } else {
+                    Row(
+                        horizontalArrangement = Arrangement.spacedBy(4.dp),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        PatchInfoChip(
+                            text = "\uD83D\uDCE6 Any package"
+                        )
+                        PatchInfoChip(
+                            text = "\uD83C\uDFAF Any version"
                         )
-                        HorizontalDivider()
+                    }
+                }
+            }
+            if (!patch.options.isNullOrEmpty()) {
+                AnimatedVisibility(visible = expandOptions) {
+                    val options = patch.options
+
+                    Column {
+                        options.forEachIndexed { i, option ->
+                            OutlinedCard(
+                                modifier = Modifier.fillMaxWidth(), colors = CardColors(
+                                    containerColor = Color.Transparent,
+                                    contentColor = MaterialTheme.colorScheme.onSurface,
+                                    disabledContainerColor = Color.Transparent,
+                                    disabledContentColor = MaterialTheme.colorScheme.onSurface
```

Comment:
These seem unnecessary if the card cannot be disabled

Reply:
These parameters are not optional/nullable